.babelrc.js

```diff
@@ -40,7 +40,6 @@ const config = {
 
 if (process.env.BABEL_ENV === "cov") {
   config.auxiliaryCommentBefore = "istanbul ignore next";
```

Comment:
jest also does this

Reply:
:) even better

Reply:
https://github.com/facebook/jest/blob/a88d3c553d94b220f9dae47f64983d9d7b3baed8/packages/jest-runtime/src/script_transformer.js#L153-L169